pages/queries.rst

```diff
@@ -45,6 +45,10 @@ Messages that do not have the field *type*::
 
   NOT _exists_:type
 
+Messages that match RegExp *FastEthernet[0-9]+* (ElasticSearch will convert messages to lowercase)::
+
+  /fastethernet[0-9]+/
```

Comment:
Either change to "/FasterEthernet[0-9]+/" here or to "fasterethernet[0-9]+" in the above paragraph.